src/main/resources/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.durabletask.executors.OnceRetentionStrategy
+
+def f = namespace(lib.FormTagLib);
+
+f.entry(title: "Idle timeout", field: "idleMinutes") {
+    f.number(default: 0)
```

Comment:
Yeah I don't use this strategy in CJOC. I suspect it needs fine-tuning to be able to run with `0` as a default